### PR TITLE
Update README.md, project.clj, and profiles.clj

### PR DESCRIPTION
**Pull request by cmhobbs**
I would normally squash these commits but it's the product of two separate branches.

Maybe we should consider environment variables for database credentials?

### CODE REVIEW

**Comment by cmhobbs** [comment]
This pullreq will cause the tests to fail because there is a function defined in `core.clj` (`db-spec`) that is a parser for the db string format that was previously used.  This will break usage of JDBC's canonical connection string format which this pullreq uses.  

I'm working on a fix.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ You will need [Leiningen][1] 2.0 or above installed.
 
 [1]: https://github.com/technomancy/leiningen
 
+### Getting up and running in Debian (wheezy)
```
Comment (hexgnu) [inline README.md:7]: Thanks for writing this out!

File: README.md [hunk 2]
```diff
@@ -12,7 +35,7 @@ To start a web server for the application, run:
 
 ## Migrations
 
-We use ragtime to handle migrations and instead of active record style migrations it utilizes SQL.
+We use clj-sql-up to handle migrations and instead of active record style migrations it utilizes SQL.
```
Comment (hexgnu) [inline README.md:38]: :)

File: profiles.clj [hunk 3]
```diff
@@ -18,7 +18,7 @@
        }
  :production 
  {
-  :env {:database-url "jdbc:postgresql://localhost/madetomeasure?user=matthewkirk"}
+  :env {:database-url "jdbc:postgresql://localhost/madetomeasure?user=deploy"}
```
Comment (hexgnu) [inline profiles.clj:21]: :boom: love it

**Comment by hexgnu** [comment]
You might need to do a merge?

File: README.md (hunk 1 above)
Comment (cmhobbs) [inline README.md:7]: No worries!  I didn't mention installation of `postgresql-jdbc-java` but I figured that would get installed with something along the way.  Given that jessie came out last week, I'll have to update this when I upgrade.  Hopefully more modern versions of the necessary packages are included.

**Comment by cmhobbs** [comment]
I'll rebase master against this branch once I solve the jdbc issues and the update the pullreq.

**Comment by cmhobbs** [comment]
closing this pullreq to replace it with a cleaned up and rebased branch.

**Pull request closed without merging**